src/relayserver/RelayServer.ts

```diff
@@ -275,6 +275,10 @@ returnValue        | ${viewRelayCallRet.returnValue}
     if (!this.isReady()) {
       throw new Error('relay not ready')
     }
+    if (this.alerted) {
```

Comment:
not really sure if this should be before all static (non-async) parameter validation, but also it doesn't really matter.